django/thunderstore/repository/views/package/_utils.py

```diff
@@ -45,3 +52,34 @@ def can_view_listing_admin(user: UserType, obj: PackageListing):
 def can_view_package_admin(user: UserType, obj: Package):
     # TODO: Object level permissions once implemented
     return user.is_staff and user.has_perm("repository.view_package")
+
+
+def get_moderatable_communities(user: Optional[UserType]) -> List[str]:
+    if not user or not user.is_authenticated:
+        return []
+
+    is_global_moderator = user.is_superuser or (
+        user.is_staff and user.has_perm("community.change_packagelisting")
+    )
+    if is_global_moderator:
+        return list(
+            (
+                str(x)
+                for x in Community.objects.order_by("pk").values_list("pk", flat=True)
+            )
+        )
```

Comment:
Going to update this syntax to use a list generator instead (as well as the one on L72)